webpack.plugins.config.js

```diff
@@ -0,0 +1,53 @@
+var webpack = require('webpack');
+var _       = require('underscore');
+
+var plugins = {
```

Comment:
It might be nicer to just export a function with configuration options rather than having to pick and choose. The way it's currently setup:
- There's a default plugins specified in common
- But, it's overwritten in several places
- Other places extend

It's a bit confusing to try to figure out which plugins are exposed where. Instead, let's just set the plugins at each end state, where config would look something like this:

```javascript
plugins: plugins({
  debug: false,
  uglify: true, // and if uglify, that means that we add the banner automatically
  // etc
})
```